docs/cri/config.md

```diff
@@ -32,6 +32,18 @@ cgroupDriver: "systemd"
 
 kubeadm users should also see [the kubeadm documentation](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).
 
+> Note: Kubernetes v1.28 supports automatic detection of the cgroup driver as
+> an alpha feature. With the `KubeletCgroupDriverFromCRI` kubelet feature gate
+> enabled, the kubelet automatically detects the cgroup driver from the CRI
+> runtime and the `KubeletConfiguration` configuration step above is not needed.gq
+>
+> When determining the cgroup driver, containerd uses the `SystemdCgroup`
+> setting from runc-based runtime classes, starting from the default runtime
+> class. If no runc-based runtime classes have been configured containerd
```

Comment:
```suggestion
> class. Starting with containerd v2.0, if no runc-based runtime classes have been configured containerd
```

Reply:
Looks good, but I don't think we will backport this to v1.6 LTS

Reply:
Thx @AkihiroSuda for quick response. I now added that separate paragraph at the end of the note. @mikebrow PTAL

Reply:
> Looks good, but I don't think we will backport this to v1.6 LTS

My reasoning to mention backport may be needed before 1.6LTS EOL:
 
The current alpha stage of this k8s feature has a feature gate that is disabled by default.. and thus the api is not required to be there for supported container runtimes (1.6, 1.7, and possibly 2.0).  The current proposed beta plan of this feature shows the feature gate is enabled by default and the api is expected to be enabled by supported container runtimes. Thus requiring clouds take the special action of disabling the feature gate if they don't move up to containerd that supports it. Then at GA stage the feature gate is expected to removed leaving the feature on and container runtimes that don't have it would cause an error, (with that current plan). Plans can always change.

our 1.6 EOL is currently set at:  max(February 15, 2025 or next LTS + 6 months) ... 

Same LTS issue with many CRI features...